ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-stable-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -48,6 +48,13 @@ releases:
       version_bounds:
         lower: 4.13.0-0
         upper: 4.14.0-0
+  source:
```

Comment:
stable -> stable does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks